src/server.ts

```diff
@@ -276,25 +276,33 @@ async function announce(
 		res.end(e.message);
 	}
 }
-
+async function statusCheck(
+	req: IncomingMessage,
+	res: ServerResponse,
+): Promise<void> {
+	res.writeHead(200);
+	res.end("OK");
+}
 async function handleRequest(
 	req: IncomingMessage,
 	res: ServerResponse,
 ): Promise<void> {
 	if (!(await authorize(req, res))) return;
```

Comment:
I could see an argument for not requiring auth but we can start like this and see what people say.

Reply:
There should be no situation when someone who _couldn't_ provide the api-key would want to check the status of cross-seed, and in the future if we add information on jobs running/scheduling etc, then we should absolutely require auth.

It can easily be passed as a param in the URL, so there's no circumstance where a service like kuma or notifarr cannot provide a ?apikey= and it be perfectly fine.

I don't want to start without an apikey now, and then add it later if we expand the return.